### PR TITLE
feature/selectable-trend-tag

### PR DESCRIPTION
**Pull request by DmitriVanGuard**
#### Summary
- Trends now can be selected;
- Selected trends are added to tags when creating new insight with `?currentTrends`;
- `-trending-words` tag is hidden while editing insight.

#### Screenshots
image

### CODE REVIEW

File: src/components/Insight/InsightEditor/InsightEditor.js [hunk 1]
```diff
@@ -31,6 +31,10 @@ class InsightEditor extends Component {
     isEditing: false
   }
 
+  trendTag = this.props.tags.find(({ name }) =>
+    name.endsWith('-trending-words')
```
Comment (Partysun) [inline src/components/Insight/InsightEditor/InsightEditor.js]: @DmitriVanGuard I think, I saw this func somewhere.

File: src/components/Trends/TrendsTable/TrendsTable.js [hunk 2]
```diff
@@ -37,9 +44,17 @@ const NumberCircle = props => (
...
 class TrendsTable extends PureComponent {
   state = {
+    selected: this.props.selectedTrends,
     connectedTrends: []
   }
 
+  componentWillUnmount () {
+    const { selected } = this.state
+    if (selected.size > 0) {
+      this.props.dispatchSelectedTrends(selected)
```
Comment (Partysun) [inline src/components/Trends/TrendsTable/TrendsTable.js]: use 'dispatch' in a name for dispatch function is not very good. Or Did I miss any trend? @DmitriVanGuard

File: src/pages/Insights/InsightCreationPage.js [hunk 3]
```diff
@@ -11,30 +11,44 @@ class InsightCreationPage extends Component {
...
+      selectedTrends,
+      ...rest
+    } = this.props
 
     if (isPublished) {
       // NOTE(vanguard): in future show thank you message
       return <Redirect to='/insights/my' />
     }
 
+    let tags
```
Comment (Partysun) [inline src/pages/Insights/InsightCreationPage.js]: can tags be undefined? @DmitriVanGuard

Comment (DmitriVanGuard) [reply src/pages/Insights/InsightCreationPage.js]: default props

File: src/components/Trends/TrendsTable/TrendsTable.js (hunk 2 above)
Comment (DmitriVanGuard) [reply src/components/Trends/TrendsTable/TrendsTable.js]: Was not able to come up with a better name

File: src/components/Insight/InsightEditor/InsightEditor.js (hunk 1 above)
Comment (DmitriVanGuard) [reply src/components/Insight/InsightEditor/InsightEditor.js]: This function was for filtering

**Pull request merged**